src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -84,7 +94,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $filenames = (array) $input->getArgument('filename');
+        $config = $input->getOption('config');
+        $excludes = $input->getOption('exclude');
         $this->format = $input->getOption('format');
+        $flags = $input->getOption('parse-tags');
+
+        if ($config) {
+            if (!file_exists($config)) {
+                throw new RuntimeException(sprintf('Lint config "%s" not found.', $config));
+            }
+
+            $lintConfig = $this->parseLintConfig($config);
+
+            if (null === $this->format && isset($lintConfig['format'])) {
+                $this->format = $lintConfig['format'];
+            }
+            if (false === $flags && isset($lintConfig['parse-tags'])) {
```

Comment:
We should probably turn the `--parse-tags` option into a negatable option so that one can override the config using either `--parse-tags` or `--no-parse-tags`.

Reply:
The negatable option is only avail with symfony/console 5.3, right?

Reply:
which is OK as you add this in symfony/yaml 5.3 too. You can bump the min symfony/console version.

Reply:
I'm not sure with the negatable: `--parse-tags` give us `true` and `--no-parse-tags` give us `false`, but the default "without the option" is also `false`. So, currently I can not detect if the user give us the option or not and the console option wins always...

We can/have to set a default `null` for this option to handle the override, but this brings a BC... Am I right?

Reply:
Setting the default value to `null` should be fine. If I don't miss anything, that shouldn't break anything.

Reply:
OK changed.